dbt/scripts/export_models.py

```diff
@@ -198,10 +200,47 @@ def main():
                 ),
             )
             table.tableStyleInfo = TableStyleInfo(
-                name="TableStyleMedium7", showRowStripes=True
+                name="TableStyleMedium11", showRowStripes=True
             )
             sheet.add_table(table)
 
+            # If a parid column exists, format it explicitly as a number to
+            # avoid Excel converting it to scientific notation when a user
+            # edits the cell
```

Comment:
**issue (blocking):** Be careful with this, I'm pretty sure Excel strips the leading zeros off of PINs when they're converted to numbers. There's a format (General > Custom > type 14 zeros) that doesn't strip leading zeros or convert to scientific notation, but I'm not sure how to use it in openpyxl.